src/dados/conteudo.js

```diff
@@ -33,4 +33,16 @@ export default [
       'música',
     ],
   },
+  {
+    nome: 'Paulo Roberto',
+    foto: 'https://media-exp1.licdn.com/dms/image/C4E03AQH6XMm5eE3AWA/profile-displayphoto-shrink_400_400/0?e=1601510400&v=beta&t=GzF2c0SKI4teGFTKkp_n4yYGZlXEB64E4c9NCutkYMk',
+    nomeDoProjeto: 'CursosCompletos',
+    url: 'https://agregadordecursos.vercel.app/',
+    tags: [
+      'agregador de cursos',
```

Comment:
Oi Paulo, tudo bom? A ideia das tags é ajudar na busca para encontrarmos algum conteúdo específico! Então pode deixar só "cursos" e, se quiser, outras tags relacionadas aos conteúdos dos cursos que for disponibilizar, pode ser?